Clarify: playlist-start/end options are inclusive (#31325)
ytdl-org/youtube-dl: README.md

                                         CIDR notation

## Video Selection:

>     --playlist-start NUMBER              Playlist video to start at (inclusive) (default is
>                                          1)

Reviewer: ```suggestion
    --playlist-start NUMBER              Playlist index of first item to include (default
                                         is 1)
```

>     --playlist-end NUMBER                Playlist video to end at (inclusive) (default is
>                                          last)

Reviewer: ```suggestion
    --playlist-end NUMBER                Playlist index of last item to include (default
                                         is last)
```
